src/Template.php

```diff
@@ -107,11 +107,17 @@ public function getParent(array $context)
         return $this->parents[$parent];
     }
 
+    /**
+     * @return self|TemplateWrapper|string|false The parent template or false if there is no parent
+     */
     protected function doGetParent(array $context)
     {
         return false;
     }
 
+    /**
+     * @return bool
+     */
     public function isTraitable()
```

Comment:
As this class is internal, I think we can use a proper tye hint:

```suggestion
    public function isTraitable(): bool
```